src/components/footer/footer--big.njk

```diff
@@ -68,7 +68,7 @@
             {% for item in secondary_section.contact_links.social_nav %}
               <div class="grid-col-auto">
                 <a class="usa-social-link usa-social-link--{{ item | lower }}" href="javascript:void(0);">
-                  <span>{{ item }}</span>
+                  <span class="usa-social-link__icon usa-social-link__icon--{{ item | lower }}" role="img" aria-label="{{ item }}"></span>
```

Comment:
Not sure if it explains what you're seeing in https://github.com/uswds/uswds/pull/4211#issuecomment-876464459, but should this be pointing to the property?

```suggestion
                  <span class="usa-social-link__icon usa-social-link__icon--{{ item.name | lower }}" role="img" aria-label="{{ item.name }}"></span>
```

Reply:
As above, this would fix the issue — but the bigger problem is that this code should match the code from `footer.njk` and use an regular image, not a background image. cc @mejiaj 

```suggestion
```

Reply:
Updated in 002a6cb.